dali/imgcodec/parsers/jpeg.cc

```diff
@@ -66,6 +72,18 @@ ImageInfo JpegParser::GetInfo(ImageSource *encoded) const {
       auto nchannels = stream->ReadOne<uint8_t>();
       info.shape = {height, width, nchannels};
       read_shape = true;
+    } else if (marker == app1_marker) {
+      DALI_ENFORCE(stream->ReadOne<jpeg_exif_header_t>() == exif_header, "Invalid EXIF header");
```

Comment:
question: Is this supposed to be the only valid header within that app1 marker?

Reply:
Oh, nice catch! I'll fix that in a moment

Reply:
Fixed in https://github.com/NVIDIA/DALI/pull/4073/commits/f2147abf0883d45684a581436373491ad4d9e6e4